pkg/server/sandbox_run.go

```diff
@@ -150,6 +176,15 @@ func (c *criContainerdService) RunPodSandbox(ctx context.Context, r *runtime.Run
 		sandbox.IP = ip
 	}
 
+	trusted := c.privilegedSandbox(r)
+	containerRuntime := c.getRuntime(trusted)
+
+	if sandbox.Config.Annotations == nil {
+		sandbox.Config.Annotations = make(map[string]string)
+	}
+
+	sandbox.Config.Annotations[annotations.PrivilegedSandbox] = fmt.Sprintf("%v", trusted)
```

Comment:
let's remove the above alloc and move this assignment to the bottom of generateSandboxContainerSpec() along with the other g.AddAnnotation() calls..

Reply:
I don't think we should change the configuration we get from CRI. We want it to reflect what we get from kubelet.

Just treat the sandbox as trusted when `Annotations` is nil.

Reply:
I think I remember hearing them say they are using this annotation.. but you make a good point we won't be using the privileged runtime unless it's privileged so if they are called it's unnecessary to have the extra flag. Maybe at one point they were being called for both privileged and unprivileged cases and needed the flag to propagate?